test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -1705,3 +1705,739 @@
                    :deferrable? "NO"},
                   :same nil}]}
                (diff-schema-maps before-migration (schema-info-map *db*))))))))
+
+(deftest migration-81-resource-events-declarative-partitioning
+  (testing "resource_events table declarative partitioning migration"
+    (jdbc/with-db-connection *db*
+      (clear-db-for-testing!)
+      (fast-forward-to-migration! 80)
+      (let [before-migration (schema-info-map *db*)]
+        (apply-migration-for-testing! 81)
+        (clojure.pprint/pprint (diff-schema-maps before-migration (schema-info-map *db*)))
+        (is (= {:index-diff
```

Comment:
Still evaluating whether these are the correct changes to expect.

Reply:
Ah, and I need to generate the expected diff a bit more dynamically to deal with the fact that the schema generates with current dates.